package.json

```diff
@@ -15,15 +14,17 @@
     "build:ssr": "yarn workspace ssr build",
     "clean": "rm -fr client/build && rm -fr ssr/dist && rm -fr content/_all-titles.json",
     "prebuild": "yarn build:client && yarn build:ssr && node content build --ensure-titles",
-    "start": "yarn run prebuild && nf start"
+    "start": "yarn run prebuild && nf start",
+    "start:functional": "cross-env ENV_FILE=testing/.env yarn run prebuild && nf -e testing/.env start"
```

Comment:
What do you think about naming this command `start:testing` instead?

Reply:
I just noticed @Gregoor's comment on the name of this command.

Reply:
No, because "testing" means too much. The `testing` yarn workspace is about all sorts of testing. This particular script/command line is about starting the usual dev environment by exclusively about functional test (aka. fixture) content exclusively. If this command was for running linting, end-to-ends, unit tests, etc. then I think `start:testing` would make more sense.